chrome/content/options_general.xul

```diff
@@ -231,7 +234,20 @@
 						        oncommand="window.open('chrome://dkim_verifier/content/treeviewSignersDefault.xul', 'SignersDefaultRules', 'chrome,resizable,centerscreen,width=600,height=300');"/>
 						<spacer/>
 					</hbox>
-				</vbox>
+					<hbox align="center">

```

Comment:
Note that this unsigned header warning is a policy independent from the sign rules, meaning if should not get disabled if `signRules.enable` is disabled.

So the new pref should not be in this indented vbox, and also the change in `chrome/content/options_general.js` is not needed.

Reply:
Thank you for the hint, I've changed it.